src/cloud-element-templates/properties-panel/properties/custom-properties/index.js

```diff
@@ -217,4 +217,9 @@ function getDefaultType(property) {
   }
 }
 
+function isEdited(isEditedEntry, property) {
+
+  // if element template has default value, it is not considered edited
+  return getDefaultValue(property) ? false : isEditedEntry;
```

Comment:
The logic should be `if property value is the same as default value, it is not considered edited`

+ we need always return a callback